python/ray/tune/trainable.py

```diff
@@ -6,7 +6,7 @@
 class Trainable(object):
     """Interface for trainable models, functions, etc.
 
-    Implementing this interface is required to use ray.tune's full
+    Implementing this interface is required to use Ray.tune's full
```

Comment:
NIT: Can you also add two quotes (\``) around L13? Looks nice on the docs.

Reply:
Done